pstress/pstress-run.sh

```diff
@@ -1574,8 +1578,30 @@ pquery_test(){
         sleep 2
       fi
     fi
-    echoit "Killing mysqld server..."
-    (sleep 0.2; kill -9 ${MPID} >/dev/null 2>&1; timeout -k5 -s9 5s wait ${MPID} >/dev/null 2>&1) &  # Terminate mysqld
+
+    if [ ${SIGNAL} == "RANDOM" ]; then
```

Comment:
decrease the duplication of code. 
"killing msyqld server using" appears  4 times.